TypeScript for Typesense?

Hello, how do I get typescript support when importing this library? It isn't critical but would be nice.

I gave it a try and ran into this error:

node_modules/typesense/lib/Typesense/Configuration.d.ts:31:16 - error TS2503: Cannot find namespace 'logger'.

31     logLevel?: logger.LogLevel;


At the top of Configuration.ts, should it be:
import * as logger from 'loglevel'

instead of right now:
import logger from 'loglevel'

The default export of loglevel doesn't seem to be the right thing.

--- a/src/Typesense/Configuration.ts
+++ b/src/Typesense/Configuration.ts
@@ -1,4 +1,4 @@
-import logger from 'loglevel'
+import * as logger from 'loglevel'
 import { MissingConfigurationError } from './Errors'
 
 export interface NodeConfiguration {
@@ -32,7 +32,7 @@ export interface ConfigurationOptions {
   useServerSideSearchCache?: boolean
   cacheSearchResultsForSeconds?: number
 
-  logLevel?: logger.LogLevel
+  logLevel?: logger.LogLevelDesc
   logger?: any //todo
 }
 

